ext/simplexml/tests/012.phpt

```diff
@@ -14,8 +14,12 @@ EOF;
 
 $sxe = simplexml_load_string($xml);
 
+try {
+    $sxe[""] = "warning";
```

Comment:
```suggestion
    $sxe[""] = "value";
```
Calling these "warning" and "error" is a bit weird if both are exceptions now :D

Reply:
haha, nice catch! :)